src/config/locales.md

```diff
@@ -22,4 +22,9 @@ Some programs have their translations in a separate package that must be
 installed in order to use them. You can
 [search](../xbps/index.md#finding-files-and-packages) for the desired language
 (e.g. "german" or "portuguese") in the package repositories and install the
-packages relevant to the applications you use.
+packages relevant to the applications you use. An especially relevant case is
+when installing individual packages from the LibreOffice suite, such as
+`libreoffice-writer`, which require installing at least one of the
+`libreoffice-i18n-**` packages to work properly. This isn't necessary when
+installing `libreoffice` directly, since it already depends on the most common
```

Comment:
"installing the `libreoffice` meta-package" is clearer, i think.

Reply:
My overall preference is to not talk of meta-package X "depending on" package Y as a way of indicating that installing X will result in the installation of Y - it feels to me like it's exposing implementation information, rather than describing the outcome from the user's perspective. So in this instance, i'd prefer something like:

> installing the `libreoffice` meta-package, since doing so will install the most common translation packages.

Reply:
Thanks!